### PR TITLE
Resolve helicopter problem of airdroppingCCC

### PR DESCRIPTION
**Pull request by HoOngEe**

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
-  "accounts_url": "https://husky.codechain.io/explorer/api/accounts" ,
+  "accounts_url": "http://localhost:9001/api/account" ,
```
Comment (majecty) [inline config/default.json]: Please do not include the diffs of the `config/default.json`

**Review by majecty — CHANGES_REQUESTED** [review]

File: config/default.json (hunk 1 above)
Comment (seonpyoKim) [reply config/default.json]: Fixed

File: index.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import { BigNumber } from "bignumber.js";
+import { H160 } from "codechain-primitives/lib/value/H160";
```
Comment (sgkim126) [inline index.ts]:
```ts
import { H160, H256, U64 } from "codechain-primitives";
```

**Review by sgkim126 — CHANGES_REQUESTED** [review]

File: index.ts (hunk 2 above)
Comment (seonpyoKim) [reply index.ts]: Fixed

**Review by majecty — APPROVED** [review]

**Review by sgkim126 — APPROVED** [review]

**Pull request merged**